examples/wallet-export/README.md

```diff
@@ -0,0 +1,52 @@
+# Example: `key-export`
+
+This example shows a private key export flow. It contains a NextJS app with:
+
```

Comment:
Do you think there's a way to consolidate both wallet and key export in a single example? I worry about having to maintain 2 demos doing roughly the same thing

(the title should say `wallet-export` and "shows a wallet export flow" if we wanted to stick with separate demos!)

What I have in mind is 2 tables instead of one, or 2 pages even. Most of the plumbing should be the same so it'll be nice to have a consolidated example.